docs/options.md

```diff
@@ -377,7 +377,7 @@ We recognize this isn’t ideal and will be revisiting this in Babel v8.
 
 Type: `boolean`
 
-You may also target browsers supporting ES Modules (https://www.ecma-international.org/ecma-262/6.0/#sec-modules). When specifying this option, the browsers field will be ignored. You can use this approach in combination with `<script type="module"></script>` to conditionally serve smaller scripts to users (https://jakearchibald.com/2017/es-modules-in-browsers/#nomodule-for-backwards-compatibility).
+You may also target browsers supporting ES Modules (<https://www.ecma-international.org/ecma-262/6.0/#sec-modules>). When specifying this option, the browsers field will be ignored. You can use this approach in combination with `<script type="module"></script>` to conditionally serve smaller scripts to users (<https://jakearchibald.com/2017/es-modules-in-browsers/#nomodule-for-backwards-compatibility>).
```

Comment:
These changes were done by markdownlint, not sure if you are ok with them? I do recommend using markdownlint :)

Reply:
Reasons why angle brackets are good: https://github.com/updownpress/markdown-lint/blob/master/rules/034-no-bare-urls.md

Reply:
I don't have strong opinion, but it will be great if we can consistently format the markdown either ways.